pkgs/desktops/mate/libmateweather/default.nix

```diff
@@ -22,17 +24,24 @@ stdenv.mkDerivation rec {
     sha256 = "VUNz3rWzk7nYSydd0spmyaSi0ObskgRPq4qlPjAy0rU=";
   };
 
-  strictDeps = true;
+  patches = [
+    # https://github.com/mate-desktop/libmateweather/pull/133
+    ./libsoup_3_support.patch
+  ];
+
+  strictDeps = false;
```

Comment:
Do you have the error that requires this?

Reply:
https://termbin.com/d37o

Reply:
```
./configure: line 15597: MATE_COMPILE_WARNINGS: command not found
checking for gtk+-3.0 >= 3.22.0... yes
checking for libxml-2.0 >= 2.6.0... no
configure: error: Package requirements (libxml-2.0 >= 2.6.0) were not met:

No package 'libxml-2.0' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBXML_CFLAGS
and LIBXML_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

That's easy, try adding libxml2 to buildInputs

Reply:
Right, `libsoup_3` no longer propagates `libxml2`. So this was a bug in `libmateweather` packaging that it relied on propagation for a direct dependency.

Reply:
so what exactly does this mean? Set strictDeps to false, or put libxml2 in buildInputs?